### ISSUE TITLE
AttributeError: 'str' object has no attribute 'decode'

### ISSUE THREAD

**Issue by sztomi** [meuter/citizenshell#10]
I'm getting the following error when using a shell constructed from a `ssh://....` string by calling `Shell`:

```
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/home/tamas/.pyenv/versions/3.6.4/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/home/tamas/.pyenv/versions/3.6.4/lib/python3.6/site-packages/citizenshell/streamreader.py", line 17, in run
    line = line.decode('utf-8').rstrip("\n\r")
AttributeError: 'str' object has no attribute 'decode'
```

If I comment out this line: https://github.com/meuter/citizenshell/blob/master/citizenshell/streamreader.py#L17

(or just remove the `decode` part) it works without an issue. 

My Python version is 3.6.4. I believe this is a Python2-Python3 incompatibility.

**Comment by meuter** [comment]
Hi, thanks for the report, I'll take a look at this week-end

**Comment by meuter** [comment]
Fixed in 2.2.1 (freshly uploaded on pypi). Feel free to reopen if you still encounter this issue.

**Comment by sztomi** [comment]
Awesome, thank you!
